commands/command-docs.md

```diff
@@ -7,7 +7,7 @@ The reply includes a map for each returned command.
 The following keys may be included in the mapped reply:
 
 * **summary:** short command description.
-* **since:** the Redis version that added the command.
+* **since:** the Redis/module version that added the command.
```

Comment:
For those who don't use modules, this can be a bit confusing. How about..

```suggestion
* **since:** the Redis version that added the command (or for module commands, the module version).
```